kinto_redis/storage.py

```diff
@@ -138,13 +138,9 @@ def _bump_timestamp(self, collection_id, parent_id, record=None,
 
     @wrap_redis_error
     def create(self, collection_id, parent_id, record, id_generator=None,
-               unique_fields=None, id_field=DEFAULT_ID_FIELD,
+               id_field=DEFAULT_ID_FIELD,
                modified_field=DEFAULT_MODIFIED_FIELD,
                auth=None):
-        self.check_unicity(collection_id, parent_id, record,
-                           unique_fields=unique_fields, id_field=id_field,
-                           for_creation=True)
-
```

Comment:
We don't need retro-compat for this plugin since previous version are buil-in previous version of Kinto,